setup.py

```diff
@@ -6,12 +7,14 @@
 import tarfile
 import glob
 
-from setuptools import setup, find_packages, Extension
-from setuptools.command.test import test as TestCommand
+from pkg_resources import get_build_platform
+from setuptools import Extension
+from setuptools import find_packages
+from setuptools import setup
 from setuptools.command.build_ext import build_ext as BuildExtCommand
 from setuptools.command.build_py import build_py as BuildPyCommand
-from pkg_resources import get_build_platform
-from distutils.command.clean import clean as CleanCommand
+from setuptools.command.test import test as TestCommand
```

Comment:
are any of these import changes necessary?

Seems like the `LIBDDWAF_VERSION` line if the only thing needed?

Reply:
YES, but those are changes done by the formatters we are usually using to sort imports and ensure PEP8. I kept them. If you really want, I can remove them.

Reply:
yeah, if you can remove them since they are unrelated to this change. we could do a separate PR to fix/update import order for `setup.py`. thanks!